packages/server/tests/acceptance/rpc_batch1.spec.ts

```diff
@@ -544,6 +544,70 @@ describe('@api-batch-1 RPC Server Acceptance Tests', function () {
         Assertions.block(blockResult, mirrorBlock, mirrorTransactions, expectedGasPrice, false);
       });
 
+      it('should not cache "latest" block in "eth_getBlockByNumber" ', async function () {
+        const blockResult = await relay.call(
+          RelayCalls.ETH_ENDPOINTS.ETH_GET_BLOCK_BY_NUMBER,
+          ['latest', false],
+          requestId,
+        );
+        await new Promise((r) => setTimeout(r, 1000));
```

Comment:
I remember there's like a .sleep() or .wait() function for these kinda waits could you please double check if not please add a util function for these waits.